gateway/api/views.py

```diff
@@ -890,3 +904,32 @@ def retrieve(self, request, pk=None):
                 instance, context={"user": user}
             )
         return Response(serializer.data)
+
+    @action(methods=["GET"], detail=False)
```

Comment:
The output should be the same as the `retrieve` end-point so in this case detail must be True.

Reply:
The doesn't work with a single object so I think the `detail` must be `False`. I verified that the exact responses are returned as the retrieve method.

Reply:
And if we don't have `detail` at all? Like, only the `GET` parameter.

Reply:
The `detail` is a required parameter for the `action` decorator.  It must be there.